java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -322,6 +324,7 @@ private StringBuilder getOutputForSingleNumber(
     } catch (NumberParseException e) {
       output.append(e.toString());
```

Comment:
As discussed, also escaped the exception message since it's output to HTML. In fact, added escaping even to output of libphonenumber when the result is printed to HTML.